backend/de.metas.async/src/main/sql/postgresql/system/42-de.metas.async/5620590_sys_gh12221_AddProcessForOpeningPdfFileFromAttachement.sql

```diff
@@ -0,0 +1,31 @@
+
+-- 2022-01-05T11:52:28.435955700Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process (AccessLevel,AD_Client_ID,AD_Org_ID,AD_Process_ID,AllowProcessReRun,Classname,CopyFromProcess,Created,CreatedBy,EntityType,IsActive,IsApplySecuritySettings,IsBetaFunctionality,IsDirectPrint,IsFormatExcelFile,IsNotifyUserAfterExecution,IsOneInstanceOnly,IsReport,IsTranslateExcelHeaders,IsUseBPartnerLanguage,LockWaitTimeout,Name,PostgrestResponseFormat,RefreshAllAfterExecution,ShowHelp,Type,Updated,UpdatedBy,Value) VALUES ('7',0,0,584956,'N','de.metas.async.process.C_Async_Batch_DownloadFileFromAttachment','N',TO_TIMESTAMP('2022-01-05 13:52:28','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.async','Y','N','Y','N','Y','N','N','N','Y','Y',0,'Download-Datei','json','N','N','Java',TO_TIMESTAMP('2022-01-05 13:52:28','YYYY-MM-DD HH24:MI:SS'),100,'C_Async_Batch_DownloadFileFromAttachment')
```

Comment:
maybe better call it "Download PDF Anhang" or "PDF Anhang"? / (EN: "Download PDF Attachment") ?
Also i think you should add a DE and EN description that tells the user that this downloads the *first* attachment that has type *PDF*.